### PR TITLE
feat(documentation): add visual regression tests for the switch component

### PR DESCRIPTION
**Pull request by davidritter-dotcom**

### CODE REVIEW

File: packages/documentation/src/stories/components/switch/switch.snapshot.stories.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+};
+
+type Story = StoryObj;
+
+export const Switch: Story = {
+  render: (_args: Args, context: StoryContext) => {
+    const templateVariants = bombArgs({
+      labelPosition: ['both', 'before', 'after'],
+      labelBefore: ['Off', 'this is longer text'],
+      labelAfter: ['On', 'this is longer text'],
```
Comment (b1aserlu) [inline packages/documentation/src/stories/components/switch/switch.snapshot.stories.ts:18]:
```suggestion
    const longerText = 'this is longer text';
    const templateVariants = bombArgs({
      labelPosition: ['both', 'before', 'after'],
      labelBefore: ['Off', longerText],
      labelAfter: ['On', longerText],
```

File: packages/documentation/src/stories/components/switch/switch.snapshot.stories.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      hiddenLabel: [false],
+      checked: [false, true],
+      disabled: [false, true],
+      validation: ['null', 'is-valid', 'is-invalid'],
+    }).map((args: Args) => {
+      return html`
+        <div class="col-6 p-3">
+          ${meta.render?.({ ...context.args, ...Default.args, ...args }, context)}
```
Comment (b1aserlu) [inline packages/documentation/src/stories/components/switch/switch.snapshot.stories.ts:26]: you might not need Default.args at all then you would only need to import meta

File: packages/documentation/src/stories/components/switch/switch.snapshot.stories.ts [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  render: (_args: Args, context: StoryContext) => {
+    const templateVariants = bombArgs({
+      labelPosition: ['both', 'before', 'after'],
+      labelBefore: ['Off', 'this is longer text'],
+      labelAfter: ['On', 'this is longer text'],
+      hiddenLabel: [false],
+      checked: [false, true],
+      disabled: [false, true],
+      validation: ['null', 'is-valid', 'is-invalid'],
+    }).map((args: Args) => {
```
Comment (b1aserlu) [inline packages/documentation/src/stories/components/switch/switch.snapshot.stories.ts:23]: Maybe you could add a filter that filters out all variations with altered label text except one where both for one or three if you want all combinations.

**Comment by gfellerph** [comment]
Closing as duplicate

**Pull request closed without merging**